Cargo.toml

```diff
@@ -119,7 +119,10 @@ url = "2.5"
 derive_more = "0.99.17"
 
 ## serde
-serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
+serde = { version = "1.0", default-features = false, features = [
```

Comment:
unrelated, format only?

Reply:
Reverted the formatting to not include unrelated diff